apps/web/test/lib/getSchedule.test.ts

```diff
@@ -392,16 +392,17 @@ describe("getSchedule", () => {
       expect(scheduleForDayWithOneBooking).toHaveTimeSlots(
         [
           // "04:00:00.000Z", - This slot is unavailable because of the booking from 4:00 to 4:15
-          "04:45:00.000Z",
-          "05:30:00.000Z",
-          "06:15:00.000Z",
-          "07:00:00.000Z",
-          "07:45:00.000Z",
-          "08:30:00.000Z",
-          "09:15:00.000Z",
-          "10:00:00.000Z",
-          "10:45:00.000Z",
-          "11:30:00.000Z",
+          `04:15:00.000Z`,
+          `05:00:00.000Z`,
+          `05:45:00.000Z`,
+          `06:30:00.000Z`,
+          `07:15:00.000Z`,
+          `08:00:00.000Z`,
+          `08:45:00.000Z`,
+          `09:30:00.000Z`,
+          `10:15:00.000Z`,
+          `11:00:00.000Z`,
+          `11:45:00.000Z`,
```

Comment:
What's the reason for this time shift by 30mins? Was it a bug?

Reply:
So we used to block out the slots for the duration of the slots - so if there was a 10m meeting at 4:15 and the event length was 45m - the next open slot would be 5:00. We've now changed this, so it will open up the nearest :15 - so 4:30 will open up again instead of 5:00.